documentation/markdown/README.md

```diff
@@ -33,6 +38,11 @@ the [changelog](https://github.com/CommunitySolidServer/CommunitySolidServer/blo
 
  * [How the server uses dependency injection](dependency-injection.md)
  * [What the architecture looks like](architecture.md)
+
+## Technical topics
+
+ * [How does authorization work](authorization.md)
+ * [How do Resource Stores work](resource-store.md)
```

Comment:
These are already linked from the `architecture.md` above. Would not link them directly from here as that way context might be missing.

Looking at `architecture.md` the alignment of the links at the bottom is wrong though so feel free to fix that while you're at it.